test/functional/legacy/increment_spec.lua

```diff
@@ -720,4 +739,10 @@ describe('Ctrl-A/Ctrl-X on visual selections', function()
       eq({}, nvim.get_vvar('errors'))
     end)
   end
+
+  it('does not drop leading zeroes', function()
+    execute('set nrformats&vi') -- &vi makes Vim compatible
```

Comment:
Does this actually do anything? I carried it over, but cant see what it should do on nvim... `:h nrformats` wasn't helpful on vim as well, so if anyone could chime in here, that would be sweet :)